### DISCUSSION TITLE
HyperLink not disappearing

### DISCUSSION THREAD

**Discussion by ashvinnihalani** [ghostty-org/ghostty#4010] [Q&A]
Once I mouse over a hyperlink, the tooltip never seems to go away and now obscures the last line of my terminal. Is there a way to force close the hyperlink tooltip

**Comment by 00-kat** [comment]
I can't seem to be able to reproduce this (on Linux).

**Comment by JakeWharton** [comment]
Dupe of https://github.com/ghostty-org/ghostty/discussions/3792. I believe there's a PR to improve this behavior already.
